.docker/Dockerfile-template.ckan

```diff
@@ -0,0 +1,32 @@
+FROM openknowledge/ckan-dev:@CKAN_VERSION@
+
+ARG SITE_URL=http://ckan:5000/
+ENV PYTHON_VERSION=@PYTHON_VERSION@
+ENV CKAN_SITE_URL="${SITE_URL}"
+
+WORKDIR "${APP_DIR}"
+
+ENV DOCKERIZE_VERSION v0.6.1
+RUN apk add --no-cache build-base \
+    && curl -sLO https://github.com/jwilder/dockerize/releases/download/${DOCKERIZE_VERSION}/dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz \
```

Comment:
if you could include the dockerize version in git it may go even faster.

Reply:
Actually, I don't think we use dockerize for anything except a socket test that could be done with preinstalled netcat.